packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -529,6 +529,8 @@ void refreshPreviewCaptureSession(
       onErrorCallback.onError("cameraAccess", "Camera is closed: " + e.getMessage());
     } catch (CameraAccessException e) {
       onErrorCallback.onError("cameraAccess", e.getMessage());
+    } catch (IllegalArgumentException e) {
+      onErrorCallback.onError("cameraAccess", e.getMessage());
```

Comment:
Based on the [documentation](https://developer.android.com/reference/android/hardware/camera2/CameraCaptureSession#setRepeatingRequest(android.hardware.camera2.CaptureRequest,%20android.hardware.camera2.CameraCaptureSession.CaptureCallback,%20android.os.Handler)) for `setRepeatingRequest`, it looks like this exception will be thrown if there's an issue with `Surface` configuration, which would suggest an internal implementation (`Camera.java`) issue, so I'm not sure we would consider this a camera error, but rather an issue we'd need to investigate.

Are you encountering this issue https://github.com/flutter/flutter/issues/117945?

Reply:
Yes. We are encountering this issue https://github.com/flutter/flutter/issues/117945.